### DISCUSSION TITLE
Series Year

### DISCUSSION THREAD

**Discussion by MichaelFitzurka** [anansi-project/comicinfo#18] [Q&A] [answered]
Taking as an example: Doom Patrol (https://comicvine.gamespot.com/search/?q=doom+patrol)
There are several series called "Doom Patrol" - starting in 1964, 1987, 2001, 2004, 2009, & 2016.
Many spanning multiple years and each with their own issue numbering.

I assume the "SeriesGroup" for all of these would just be "Doom Patrol".

The "Year" value in the schema is for publishing year, so it doesn't identify alone which series it comes from.

Adding a "SeriesYear" to differentiate between multiple series with the same name could work.

Is there a different way to represent this distinction within the existing schema?
AlternateSeries of "Doom Patrol (1964)"?

**Comment by SenorSmartyPants** [comment, marked as answer]
This is typically stored in `Volume`. `SeriesGroup` would not be necessary. This is how ComicRack and ComicVine Scraper handle this.

**Comment by FibreTTP** [comment]
Could I please get confirmation that `Year` is intended for the release year of the book, and not the series itself?

Wondering whether the year of series publication should be included as the `Year` in each book of a series. See https://github.com/mansuf/mangadex-downloader/pull/175:

> While Kavita [implies](https://wiki.kavitareader.com/guides/metadata/comics) that only using the year means that the year defines the release date of the _series_, the official schema [states](https://anansi-project.github.io/docs/comicinfo/documentation) that the all of the year, month, and day, are used as the release date of the _book_ (i.e., volume, chapter -- whatever the `.cbz` file contains). (the `year` returned by MangaDex is of the _series_.)

**Reply by gotson** [reply]
it's the book's release date. The year of the series publication is not stored in ComicInfo, but can be inferred from the release date of the first book in the series.
